### DISCUSSION TITLE
Space Focus jumping to displays

### DISCUSSION THREAD

**Discussion by jin-ahn** [asmvik/yabai#1673] [Q&A] [answered]
I have multiple displays and mulitple spaces. I have a short to just move between spaces as shown below. What I want/expect to happen is that when I reached my last space, I would be unable to "--focus next" since there is not a next and vice versa for prev. However what I am noticing that what ends up happen when I get to my last space and use the shortcut to focus next, it will jump focus to my other display. I'm assuming this is happening because yabai sees the space on my other display as the "next" space. 

**_TLDR_**
**Is there way a to configure it where  "-m space --focus next" only concerns itself with the spaces assigned to the current display rather than spaces across all displays?**

shift + alt - u: yabai -m space --focus next 
shift + alt - y: yabai -m space --focus prev

**Comment by asmvik** [comment, marked as answer]
Replying here for searching purposes; copied from #1771 

--

There is no such config (and I am not interested in adding one), however as I described in #1765 which is pretty much the same concern, you can use the following binds/commands instead to achieve what you are looking for: 

```
# focus next space, if the current space is not the last space on its display
shift + alt - u : if [[ $(yabai -m query --spaces --display | jq '.[-1]."has-focus"') == "false" ]]; then yabai -m space --focus next; fi

# focus prev space, if the current space is not the first space on its display
shift + alt - y : if [[ $(yabai -m query --spaces --display | jq '.[0]."has-focus"') == "false" ]]; then yabai -m space --focus prev; fi
```
